.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+🚨HOLD UP PLEASE 🚨
```

Comment:
There's a space between the emoji and the text at the end but not at the start :eyes: (sorry)